bridge/bridgeapp/templates/registration/login.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'home.html' %} {% block main %}
+<section>
+    {% if form.errors %}
+    <p>The input is not valid.</p>
+    {% endif %}
+    
+    {% if next %} 
+    <p>You do not have permission to access that page.</p>
+    {% endif %} 
+
+    <form method="post" action="{% url 'login' %}">
+        {% csrf_token %} 
+
+        <p>Username: {{ form.username }} </p>
```

Comment:
Consider using `<label>Username: {{ ...}}</label>` to improve accessibility. That way screen readers will know what the input is for.

Reply:
Agree. And we can even use the `visually-hidden` class of Bootstrap to accommodate accessibility while using placeholder text or an icon for the label, which is commonplace these days.

Reply:
I had not thought of label's effect on focusing the text field and missed the point of _accessibility_ in talking about `visually-hidden`. Thanks Dustin.